compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -4718,44 +4568,46 @@ private BLangExpression getMatchedExprIfCalledInMatchGuard(BLangInvocation invoc
         return null;
     }
 
-    /**
-     * Experimental feature list for JBallerina 1.0.0.
-     *
-     * @since JBallerina 1.0.0
-     */
-    private enum ExperimentalFeatures {
-        LOCK("lock"),
-        XML_ACCESS("xml access expression"),
-        XML_ATTRIBUTES_ACCESS("xml attribute expression"),
-        ;
-        private String value;
-
-        private ExperimentalFeatures(String value) {
-
-            this.value = value;
-        }
-
-        @Override
-        public String toString() {
-
-            return value;
-        }
-    }
-
     private enum DefaultValueState {
         NOT_IN_DEFAULT_VALUE,
         RECORD_FIELD_DEFAULT,
         OBJECT_FIELD_INITIALIZER,
         FUNCTION_IN_DEFAULT_VALUE
     }
 
-    private static class PotentiallyInvalidAssignmentInfo {
-        List<Location> locations;
-        BLangInvokableNode enclInvokable;
-
-        private PotentiallyInvalidAssignmentInfo(List<Location> locations, BLangInvokableNode enclInvokable) {
-            this.locations = locations;
-            this.enclInvokable = enclInvokable;
-        }
+    /**
+     * @since 2.0.0
```

Comment:
Let's document this properly and fix the version also.

Reply:
Changed